mindsdb/interfaces/data_catalog/data_catalog_loader.py

```diff
@@ -204,6 +204,12 @@ def _add_column_statistics(self, df: pd.DataFrame, tables: db.MetaTables, column
                 # Convert the distinct_values_count to an integer if it is not NaN, otherwise set it to None.
                 val = row.get("distinct_values_count")
                 distinct_values_count = int(val) if pd.notna(val) else None
+                min_val = row.get("minimum_value")
+                max_val = row.get("maximum_value")
+                if min_val is not None and not isinstance(min_val, (str, int, float)):
```

Comment:
@ZoranPandovski I understand that storing int and float values in a string column will be allowed in SQLite, but I don't think will work in Postgres?